packages/dashboard/src/features/datasource/components/ApifyConnectedPanel.tsx

```
@@ -0,0 +1,198 @@
+  useApifyLatestData,
+  useApifyRuns,
+} from '#features/datasource/hooks/useDatasource';
+import {
+  datasourceService,
+  type ApifyConnection,
+} from '#features/datasource/services/datasource.service';
+
+const APIFY_CONSOLE_URL = 'https://console.apify.com';
+
+function fmtTime(iso: string | null): string {
+  if (!iso) return '—';
```

ESLint

[error] Expected { after 'if' condition.

(curly)